Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -890,8 +896,9 @@ def addVertexFitting(
     outputDirRoot: Optional[Union[Path, str]] = None,
     associatedParticles: str = "particles_input",
     trajectories: Optional[str] = None,
-    trackParameters: str = "trackparameters",
+    trackParameters: str = "fittedTrackParameters",
     vertexFinder: VertexFinder = VertexFinder.Truth,
+    trackSelectorConfig: Optional[TrackSelectorConfig] = TrackSelectorConfig(),
```

Comment:
```suggestion
    trackSelectorRanges: Optional[TrackSelectorRanges] = None,
```
also add a decorator to the `def addVertexFitting`:
```
@acts.examples.NamedTypeArgs(
trackSelectorRanges=TrackSelectorRanges
)
```
and a comment to the docstring:
```
    trackSelectorRanges : TrackSelectorRanges(absEta, pt)
        TrackSelector configuration. Each range is specified as a tuple of (min,max).
        Defaults of no cuts specified in Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/TrackSelector.hpp
```

Reply:
thanks for the suggestions! I applied all of them but taking a second look the `addSeeding` helper is also using the default construction for the config/ranges. I thought that might be good as this will be the default usage

Reply:
The advantage of having `None` as the default is that you can skip the `TrackSelector` altogether if it is not needed. This works because `TrackSelector` defaults are "don't cut any tracks".